_posts/2018-11-20-docker-setup.md

```diff
@@ -0,0 +1,336 @@
+---
```

Comment:
Make sure we keep this information in just one place

Reply:
Yep, have another PR to remove this information from the Supplejack Docker repo and link to this page